src/services/upload/uploadManager.ts

```diff
@@ -103,6 +103,12 @@ class UploadManager {
         collections: Collection[]
     ) {
         try {
+            filesWithCollectionToUploadIn.sort(
+                (aFileWithCollection, bFileWithCollection) =>
+                    (aFileWithCollection.file['path'] as string).localeCompare(
```

Comment:
Are we confident that the file['path] will be always non-null/undefined? Not sure what will be the behaviour when path is missing.